### PR TITLE
Add README & Credits

### PR DESCRIPTION
**Pull request by karst**
I have moved unused samples to its own folder so it does not clutter up the SoundFX library. I have not removed these since they are part of a set and could be used in the future if we want to add more variety to some trigger's pull list.

I have also created a credits file, since the project is more or less taking off with the addition of asset packs I thought this would be a smart addition so people can see we own all rights to the samples included.

Lastly I have updated the main README file to include the Sound Effects index so more people can contribute and test more easily.

### CODE REVIEW

File: Credits.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+train_whistle_1_cp_railroad, train_whistle_2_cp_railroad, train_whistle_3_cp_railroad, train_whistle_depart_cp_railroad, bm_rc_lift_cp_rougarou, bm_rc_roar_cp_rougarou,
+wood_rc_lift_cp_bluestreak, scream_el_zadra, hyper_rc_lift_cp_magnum_xl_200,
+
+Ted John (IntelOrca):
+crowd, 
+
+(ThatGuyYouKnow):
+hydrolics_ninja,
+
+All samples listed above are recorded for OpenRCT2 and are licensed under CC BY-SA (https://creativecommons.org/licenses/by-sa/3.0/)
```
Comment (Generalcamo) [inline Credits.txt]: The license currently in the repo is the CC BY-SA 4.0 license, not the 3.0 license. 
```suggestion
All samples listed above are recorded for OpenRCT2 and are licensed under CC BY-SA 4.0 (https://creativecommons.org/licenses/by-sa/4.0/)
```

Comment (Generalcamo) [reply Credits.txt]: As I found out, this is actually licensed under CC BY 4.0. A PR to change this is now open (#9).

**Comment by IntelOrca** [comment]
I did not source the crowd sample, I can't remember where I got it from. Possibly discord chat?

**Comment by Gymnasiast** [comment]
I merged this, but we will have to add a list of authors of each sample. And make sure to check that all samples that were not made by the people who committed them are indeed released under a compatible licence.

**Pull request merged**